.gitignore

```diff
@@ -1,9 +1,35 @@
-**/.idea/
-
 **/target
 
-**/*.iml
-
 **/dependency-reduced-pom.xml
 
 **/?
+
+!.mvn/wrapper/maven-wrapper.jar
+!**/src/main/**
+!**/src/test/**
+
+### STS ###
+**/.apt_generated
+**/.classpath
+**/.factorypath
+**/.project
+**/.settings
+**/.springBeans
```

Comment:
springBeans?

Reply:
Actually I was using Eclipse and these are for my Spring projects and just added them.

I'll get rid of factorypath and springbeans for the meantime